src/lib/electron.ts

```diff
@@ -498,4 +500,44 @@ export class SentryElectron implements Adapter {
       return originalEmit.call(emitter, event, ...args);
     };
   }
+
+  private normalizeEvent(event: any) {
+    const appPath = app.getAppPath();
+
+    if (event.culprit) {
+      event.culprit = this.normalizeUrl(event.culprit, appPath);
+    }
+
+    if (event.request && event.request.url) {
```

Comment:
Is `event.request.url` normalisation required to properly fingerprint in Sentry?

Reply:
If there is a stacktrace, the the frames of it will be used for the fingerprint, so I would keep it like this.